cmd/embedded-cluster/list_images.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli/v2"
+)
+
+var listImagesCommand = &cli.Command{
+	Name:   "list-images",
```

Comment:
how is this different from the metadata list-images command?

Reply:
its not just two different tools. metadata list-images is unused

Reply:
can it be removed then? do we need it?

Reply:
removed